core/src/OpenMPTarget/Kokkos_OpenMPTarget_Parallel.hpp

```diff
@@ -872,13 +872,15 @@ TeamThreadRange(const Impl::OpenMPTargetExecTeamMember& thread, iType count) {
       iType, Impl::OpenMPTargetExecTeamMember>(thread, count);
 }
 
-template <typename iType>
+template <typename iType1, typename iType2>
 KOKKOS_INLINE_FUNCTION Impl::TeamThreadRangeBoundariesStruct<
     iType, Impl::OpenMPTargetExecTeamMember>
-TeamThreadRange(const Impl::OpenMPTargetExecTeamMember& thread, iType begin,
-                iType end) {
+TeamThreadRange(const Impl::OpenMPTargetExecTeamMember& thread, iType1 begin,
```

Comment:
Why are these free function declared in the `Impl::` namespace?

Reply:
I removed the dead code.